input/profiles/StructureDefinition-cqm-populationBasis.json

```diff
@@ -66,7 +66,7 @@
         "definition": "The population basis specifies the type of elements in the population. For a subject-based measure, this is boolean (because the subject and the population basis are the same, and the population criteria define yes/no values for each individual in the population). For measures that have a population basis that is different than the subject, this element specifies the type of the population basis. For example, an encounter-based measure has a subject of Patient and a population basis of Encounter, and the population criteria all return lists of Encounters.",
         "comment": "In Health Quality Measure Format, this was referred to as the Item Count metadata attribute.",
```

Comment:
Add a description of the usage of basis for data requirements: If the populationBasis is specified as a data requirement, the type SHALL be present, profile and codeFilter MAY be present, and no other elements are allowed. This usage allows population basis to be stated with more specificity, e.g. CT Scans, rather than just DiagnosticReport.